src/main.ts

```diff
@@ -29,6 +32,26 @@ export async function main(): Promise<Outputs> {
         failOnStdErr = true
     }
 
+    let whatIf
+    try {
+        whatIf = getBooleanInput('whatIf')
```

Comment:
`whatIf` has a default value `false`. So why the try-catch?
And why not for `whatIfExcludeChangeTypes` and `whatIfResultFormat`?

Reply:
if someone adds a non boolean input then getBooleanInput will throw an error, so by using try catch we make sure its atleast false.